8.x/theme-development/articles.md

```diff
@@ -1,3 +1,407 @@
+
+### Article properties table
+
+The article is a system object in itself and can display properties on the pages of the article.
+
+On each page of the blog list, there is a series of categories after entering one of them, passing the `$article` variable from the controller to the blade. Then with `foreach` display, these article features `image`, `title`, `create_at`, and `short_content`.
+
+If you go to the article model, you can see the properties of this object. The properties of the article are shown in the table below:
+
+Num   | Property           | Datatype     | Description
+------|--------------------|--------------|------------
+**1** | `id`               | integer      | Primary key.
+**2** | `directory_id`     | integer      | The parent category.
+**3** | `system_user_id`   | integer      | Specifies which user created the article.
+**4** | `title`            | string       | Specifies the title of the article.
+**5** | `short_content`    | string       | Specifies the opening description of each article.
+**6** | `full_text`        | string       | All the content of the article or blog post is stored in it.
+**7** | `source`           | string       | The source is a short string that holds the image path as a big picture.
+**8** | `image_path`       | string       | Specifies the path of the image
+**9** | `created_at`       | DateTime     | Article creation date.
+**10**| `updated_at`       | DateTime     | Article update date
+**11**| `average_rating`   | float        | Article rating.
+**12**| `rates_count`      | int          | Number of article rates.
+**13**| `content_type`     | int          | Specifies the type of article.
+**14**| `seo_description`  | mixed        | Description of a article SEO.
+**15**| `seo_keywords`     | mixed        | SEO keywords of a article.
+**16**| `is_suggested`     | boolean      | Specifies suggested articles.
+
+**NOTE:** In Larammerce there is complete freedom of action so that the administrator can create directories as a tree and put a **product** or an **article** in each directory.
+
+For more information about the products in Larammerce, see "[Products](https://docs.larammerce.com/8.x/theme-development/products.html)".
+
+**NOTE:** `content_type` is currently static, in the future the Larammerce team intends to create it dynamically.
+
+**NOTE:** `article` is a Laravel model, extends from Laravel `BaseModel`.
+
+##### SOURCE
+
+```php
+class Article extends BaseModel implements
+```
+
+An article is related to objects, following, you can see these relationships in the table:
+
+Num  | Property       | Datatype     | Description
+-----|----------------|--------------|-----------
+**1**| `directory`    | Directory    | Specifies what category this article is in.
+**2**| `directories`  | Directory[]  | There is a list of parent directories that indicate in which other directories this article is manufactured, or (Short link of article in other directories).
```

Comment:
A list of parent categories indicates which other directories this article belongs to. (Also there are some symbolic links that are managed similarly.)